README.md

```diff
@@ -164,14 +164,12 @@ https://127.0.0.1:4443/api/delete_employee/101
 
 ## Oracle Linux VM Deploy 
 
-### 1. Install Python 3.6, flask , cx_Oracle, Jinga2 & six packages on Oracle Linux 7
+### 1. Install Python, flask , cx_Oracle, Jinga2 & six packages on Oracle Linux 7
```

Comment:
No new/recent content from Oracle should be using OL7.  As the Linux team remind us "OL7 has moved to ELS support only"